target_chains/solana/programs/pyth-price-publisher/src/processor/initialize.rs

```diff
@@ -0,0 +1,126 @@
+use {
+    crate::{
+        accounts,
+        instruction::{
+            InitializeArgs,
+            CONFIG_SEED,
+        },
+        validate::{
+            validate_config,
+            validate_payer,
+            validate_system,
+        },
+    },
+    solana_program::{
+        account_info::AccountInfo,
+        entrypoint::ProgramResult,
+        program::invoke_signed,
+        pubkey::Pubkey,
+        rent::Rent,
+        system_instruction,
+        sysvar::Sysvar,
+    },
+};
+
+// Creates a config account that stores the authority pubkey.
+// The authority is allowed to modify publisher configs.
+pub fn initialize(
```

Comment:
Can you document the expected accounts for all of these instructions? It's hard to tell now without reading the method implementation carefully. Also please document any assumptions around those accounts (E.g., this is uninitialized and becomes initialized after)

Reply:
Addressed in #1883.